legend-engine-xts-data-space/legend-engine-xt-data-space-protocol/src/test/resources/dataSpace_TestData.json

```diff
@@ -45,11 +45,13 @@
       ],
       "executables": [
         {
+          "_type": "dataSpacePackageableElementExecutable",
```

Comment:
this change shouldnt be needed. if no _type we should default to DatasoacePackagableElementExecutable to keep it backward compatible.